app/js/components/Dashboard.jsx

```diff
@@ -7,51 +7,77 @@
  * graphic logo is a trademark of OpenMRS Inc.
  */
 import React from 'react';
-import { Link } from 'react-router-dom';
 
-import openmrsLogo from '../../img/openmrs-with-title-small.png';
-import loadingGif from '../../img/loading.gif'
+export default () => {
+  const openmrsServer = window.location.pathname.split('/')[1];
+  const addonManagerLink = `/${openmrsServer}/owa/addonmanager/index.html`;
 
-export default class Dashboard extends React.Component {
-  render() {
-    const openmrsServer = window.location.pathname.split('/')[1];
-    return (
-      <div className="list-group home-list">
-        <Link to="#" className="list-group-item list-group-item-action flex-column align-items-start active">
-          <div className="d-flex w-100 justify-content-between">
-          
-          <h5 className="mb-1">The Platform is running</h5><img src={(loadingGif)} alt=""/>
-          </div>
-          
-        </Link>
-        <Link to="#" className="list-group-item list-group-item-action flex-column align-items-start" onClick={(event) => {
-          event.preventDefault();
-          window.location = `/${openmrsServer}/owa/Add On Manager/index.html#/`;
-          }}>
-          <div className="d-flex w-100 justify-content-between">
-            <h5 className="mb-1">Addon Manager</h5>
-            
-          </div>
-          <p className="mb-1">The Addon Manager open web app, is an OpenMRS tool used to manage the uploading, installation, upgrading, deleting and viewing of OpenMRS addons.</p>
-          
-        </Link>
-        <Link to="#" className="list-group-item list-group-item-action flex-column align-items-start">
-          <div className="d-flex w-100 justify-content-between">
-            <h5 className="mb-1">User Manager</h5>
-            
-          </div>
-          <p className="mb-1">To learn more about the User Interface Modules, click here for details about installing them</p>
-          
-        </Link>
-        <Link to="#" className="list-group-item list-group-item-action flex-column align-items-start">
-          <div className="d-flex w-100 justify-content-between">
-            <h5 className="mb-1">REST Documentation</h5>
-            
+  return (
+    <div id="body-wrapper">
+      <div className="container">
+        <div className="row">
+          <div className="col">
+            <h4 className="status-header">
+              OpenMRS platform is running
+              <i className="fa fa-check-circle float-right" />
+              <span />
+            </h4>
+
+            <ul className="platform-ul">
+              <div>
+                <li className="platform-list">
+                  <a
+                    href={addonManagerLink}
+                    className="platform-item-link"
+                  >
```

Comment:
Why did you switch from React Router Link to anchor tags?

Reply:
Hey @emmabaye, nice observation, however, I switched from `Link` because;-
Link uses a `to` attribute whose value will be appended at the end of the URL to produce something like: `http://localhost:8080/openmrs/owa/openmrs-owa-platformui/index.html#/http://localhost:8080/openmrs/owa/addonmanager/index.html`. 
instead of `http://localhost:8080/openmrs/owa/addonmanager/index.html` which is achieved using the anchor tag. I don't need the Link there since this is not a path registered in the BrowserRoute routes.